api/client.go

```diff
@@ -59,10 +60,13 @@ func checkError(resp *http.Response, body []byte) error {
 func ClientFromEnvironment() (*Client, error) {
 	defaultPort := "11434"
 
-	scheme, hostport, ok := strings.Cut(os.Getenv("OLLAMA_HOST"), "://")
+	hostVar := os.Getenv("OLLAMA_HOST")
+	hostVar = strings.Trim(strings.TrimSpace(hostVar), "\"'")
```

Comment:
Not sure if this is what you intended but this trims space before quotes. More common would be trim quotes then space? e.g. `OLLAMA_HOST="127.0.0.1:11434 "` vs. `OLLAMA_HOST="127.0.0.1:11434"<space>`

Reply:
Yeah, I went back and forth on this. I'm not 100% sure how Windows stuffs in the quotes. I can switch it to the other way if that makes more sense, or even do both.

Reply:
I believe the typical pattern we're seeing in windows is user-error not realizing the GUI and `cmd` shouldn't have quotes, while `powershell` requires them.  In the context of this particular env var where a space isn't valid, I think it probably makes sense to trim spaces before and after.